.github/workflows/linux_conditional_compilation.yml

```diff
@@ -40,17 +41,44 @@ jobs:
           component_pattern: "category: (.*)"
           repo_token: ${{ secrets.GITHUB_TOKEN }}
           skip_when_only_listed_labels_set: 'docs'
-          skip_when_only_listed_files_changed: '*.md,*.rst,*.png,*.jpg,*.svg,*/layer_tests_summary/*,*/conformance/*'
```

Comment:
Was it removed intentionally?

Reply:
unintentionally copied from `linux_arm64.yml`, reverted.